ecs-agent/managed_daemon/mountpoint.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package manageddaemon
+
+type MountPoint struct {
+	// SourceVolumeID is used to identify the task volume globally, it's empty
+	// when for internal mount points.
+	SourceVolumeID string `json:"SourceVolumeID"`
+	// SourceVolume is the name of the source volume, it's unique within the task.
+	SourceVolume string `json:"SourceVolume,omitempty"`
```

Comment:
nit(non-blocking): since it's the volume name, it might be more intuitive to call it something like `SourceVolumeName`

Reply:
This is actually meant to mirror the incoming containerd agent's Mountpoint definition.

Reply:
I see, thanks for clarifying.